### PR TITLE
Assign tasks

### PR DESCRIPTION
**Pull request by schromya**
+ Assign tasks to users in each project using the task modal.

### CODE REVIEW

**Review by erwilliams64 — APPROVED** [review]

File: app/views.py [hunk 1]
```diff
@@ -282,6 +282,10 @@ def taskView(request, projectID, taskID):
     @returns {HttpResponse} - An HttpResponse object that renders the taskModal.html
         template with the project ID, task ID, and task form context.
     """
+
+    if request.method != "POST" and request.method != "GET":
+        return
+
```
Comment (crazykyler005) [inline app/views.py:288]: shouldn't we be using an `or` instead of an `and` here?

File: app/views.py [hunk 2]
```diff
@@ -336,20 +361,20 @@ def taskView(request, projectID, taskID):
         
         print(form.errors)
 
-    # If a GET (or any other method) we'll create a blank form
+    # If a GET (or any other method) we'll create a form
```
Comment (crazykyler005) [inline app/views.py]:
```suggestion
    # If request method is GET we'll create a form
```

File: app/views.py [hunk 3]
```diff
@@ -291,13 +295,33 @@ def taskView(request, projectID, taskID):
...
+
+    userData, statusCode = UsersAPIView.getUsers(
+        [{"id": id} for id in projectData.get("userIDs")], getAuthorizationToken(request))
+    if statusCode != status.HTTP_200_OK:
+        print(f"Users fetch failed: {data.get('message')}")
+        return HttpResponseServerError(f"An error occurred: {data.get('message')}")
+
+    
+    assigneeChoices = [(data.get('id'), data.get('username')) for data in userData]
+
```
Comment (crazykyler005) [inline app/views.py]: `data` is already defined so we should probably change this reference variable name

```suggestion
    
    assigneeChoices = [(userData.get('id'), userData.get('username')) for userData in userDataList]

```

**Review by crazykyler005 — APPROVED** [review]
Looks good to me

File: app/views.py (hunk 1 above)
Comment (schromya) [reply app/views.py:288]: If its "or",  it will always evaluate and return.

**Pull request merged**